# cd /data/openpilot && curl https://raw.githubusercontent.com/eFiniLan/openpilot-patches/master/0.5.13/disable_driver_monitor.diff | git apply -v
#
# This patch prevents driver monitor sending warning events
# ** This pretty much disable all the safety checks/detection, use with caution **
#
===================================================================
--- a/selfdrive/controls/lib/driver_monitor.py
+++ b/selfdrive/controls/lib/driver_monitor.py
@@ -154,8 +154,8 @@
     elif self.awareness <= self.threshold_pre:
       # pre green alert
       alert = 'preDriverDistracted' if self.monitor_on else 'preDriverUnresponsive'
-    if alert is not None:
-      events.append(create_event(alert, [ET.WARNING]))
+    # if alert is not None:
+    #   events.append(create_event(alert, [ET.WARNING]))

     return events

